app/src/androidTest/java/de/danoeh/antennapod/fragment/preferences/synchronization/ViewDataProviderTest.java

```diff
@@ -0,0 +1,26 @@
+package de.danoeh.antennapod.fragment.preferences.synchronization;
+
+import junit.framework.TestCase;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.sync.SynchronizationProviderViewData;
+
+public class ViewDataProviderTest extends TestCase {
+
+    public void testGetSynchronizationProviderHeaderSummary() {
+        assertEquals(R.string.synchronization_summary_nextcloud,
+                ViewDataProvider.getSynchronizationProviderHeaderSummary(
+                        SynchronizationProviderViewData.NEXTCLOUD_GPODDER.getName()));
```

Comment:
Hmm, to me, `getName()` sounds like it is user readable text. How about calling the stored values `id` instead?